copier/tools.py

```diff
@@ -119,17 +118,6 @@ def copy_file(src_path: Path, dst_path: Path, follow_symlinks: bool = True) -> N
     shutil.copy2(src_path, dst_path, follow_symlinks=follow_symlinks)
 
 
-def to_nice_yaml(data: Any, **kwargs) -> str:
-    """Dump a string to pretty YAML."""
-    # Remove security-problematic kwargs
-    kwargs.pop("stream", None)
-    kwargs.pop("Dumper", None)
-    result = safe_dump(data, **kwargs)
-    if isinstance(result, str):
-        result = result.rstrip()
```

Comment:
Contrary to this function, the `to_nice_yaml` from Ansible filters does not seem to `rstrip()` the result. This is the reason the tests fail, but for the life of me I was not able to fix the test! I tried adding or removing newlines with multiple methods, but pytest would not have it. I'll continue trying of course, but if you want to to take a look I don't say no :sweat_smile:

Reply:
It's weird, but it doesn't seem like a big loss. You can modify the tests if you want.

Reply:
Pffff, I was tired I guess, it's just that the same assert is repeated three times and I didn't get that until now... Tests now passing. I will add some more.